### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by soulyyyyy**
Updated links in all docs (gitlab to github)

### CODE REVIEW

File: docs/use-cases/token-agnostic.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ Fundamentally, the product provides four capabilities:
 - Token swaps using liquidity from popular DEXes such as Uniswap, PancackeSwap, and TraderJoe;
 - [Rarimo's bundling technology](https://rarimo.gitlab.io/docs/docs/tutorials/bundling) allows the inclusion of custom logic into transactions. Rarimo contracts use this logic to run actions that follow an initial transaction, such as unlocking tokens on other chains and buying NFTs;
 - [Circle CCTP integration](https://rarimo.gitlab.io/docs/docs/overview/applications/usdc-integration#nft-checkout) provides USDC liquidity for all supported chains;
-
+--- rarimo.gitlab.io links above should be changed, but idk actual links. These links return 404 page not found.
```
Comment (ihordiachenko) [inline docs/use-cases/token-agnostic.mdx:11]: Replace them with https://docs.rarimo.com and re-check that they work

File: docs/use-cases/social-graphs.mdx [hunk 2]
```diff
@@ -34,7 +34,7 @@ Using such proofs also makes for identity providers minting identity quite easy
 ## Rarimo Protocol
 
 Rarimo is a cross-chain interoperability protocol for the social layer. It provides trustless oracles of on-chain events from all supported blockchains, a cost-efficient way to bridge the assets or data, and EVM-compatible smart contracts for additional flexibility.
-
+ --- bots rarimo.gitlab.io links below should be changed, but idk actual links. These links returns 404 page not found.
```
Comment (ihordiachenko) [inline docs/use-cases/social-graphs.mdx:37]: Replace them with https://docs.rarimo.com/ and re-check that they work

File: docs/how-to-guides/polygon-id-state-replication.mdx [hunk 3]
```diff
@@ -25,12 +25,14 @@ docker pull registry.gitlab.com/rarimo/polygonid/relayer-svc:cca1632497291b16a69
 573
 ```
 
-Or clone the  [repository](https://gitlab.com/rarimo/polygonid/relayer-svc) and build it from the source:
+Or clone the  [repository](https://github.com/rarimo/relayer-svc) and build it from the source:
```
Comment (ihordiachenko) [inline docs/how-to-guides/polygon-id-state-replication.mdx:28]: This is the correct repo: https://github.com/rarimo/identity-relayer-svc?tab=readme-ov-file

File: docs/how-to-guides/polygon-id-state-replication.mdx [hunk 4]
```diff
@@ -25,12 +25,14 @@ docker pull registry.gitlab.com/rarimo/polygonid/relayer-svc:cca1632497291b16a69
...
 
 <!-- TODO: fix docker image -->
 
 ```sh
-git clone git@gitlab.com:rarimo/polygonid/relayer-svc.git
+git clone git@github.com:rarimo/relayer-svc.git
+
+#after changing clone link from gitlab to github checkout should be removed probably?
```
Comment (ihordiachenko) [inline docs/how-to-guides/polygon-id-state-replication.mdx:35]: Yep, remove it

**Review by ihordiachenko — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/polygon-id-state-replication.mdx (hunk 4 above)
Comment (soulyyyyy) [reply docs/how-to-guides/polygon-id-state-replication.mdx:35]: also, git clone should be from git@github.com:rarimo/identity-relayer-svc.git and not from git@github.com:rarimo/relayer-svc.git, yes?

Comment (ihordiachenko) [reply docs/how-to-guides/polygon-id-state-replication.mdx:35]: Yep. rarimo/relayer-svc is for bridging tokens, not for the Polygon ID replication

File: docs/use-cases/social-graphs.mdx [hunk 5]
```diff
@@ -34,8 +34,8 @@ Using such proofs also makes for identity providers minting identity quite easy
 ## Rarimo Protocol
 
 Rarimo is a cross-chain interoperability protocol for the social layer. It provides trustless oracles of on-chain events from all supported blockchains, a cost-efficient way to bridge the assets or data, and EVM-compatible smart contracts for additional flexibility.
-
-A combination of [threshold signature technology(TSS)](https://rarimo.gitlab.io/docs/docs/developers/tss/overview), instant finality, [EVM transaction bundling](https://rarimo.gitlab.io/docs/docs/tutorials/bundling), and low protocol fees make cross-chain interactions affordable and hassle-free for the end user.
+ --- bots rarimo.gitlab.io links below should be changed, but idk actual links. These links returns 404 page not found.
```
Comment (ihordiachenko) [inline docs/use-cases/social-graphs.mdx:37]: Clean up your comments

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**